src/test/resources/alma-fix/990110486750206441.json

```diff
@@ -0,0 +1,245 @@
+    } ]
+  }, {
+    "type" : [ "ComplexSubject" ],
+    "label" : "Solingen | Wissenschaftler",
+    "componentList" : [ {
+      "type" : [ "PlaceOrGeographicName" ],
+      "label" : "Solingen",
+      "source" : {
+        "label" : "Gemeinsame Normdatei (GND)",
+        "id" : "https://d-nb.info/gnd/7749153-1"
+      },
+      "id" : "https://d-nb.info/gnd/4055431-4",
+      "gndIdentifier" : "4055431-4",
+      "altLabel" : [ "Stadt Solingen", "Klingenstadt Solingen", "Salingiacum" ]
+    }, {
+      "type" : [ "SubjectHeading" ],
+      "label" : "Wissenschaftler",
+      "source" : {
+        "label" : "Gemeinsame Normdatei (GND)",
+        "id" : "https://d-nb.info/gnd/7749153-1"
+      },
+      "id" : "https://d-nb.info/gnd/4066567-7",
+      "gndIdentifier" : "4066567-7"
+    } ]
+  }, {
+    "type" : [ "ComplexSubject" ],
+    "label" : "Entomologe",
+    "componentList" : [ {
+      "type" : [ "SubjectHeading" ],
+      "label" : "Entomologe",
+      "source" : {
+        "label" : "Gemeinsame Normdatei (GND)",
+        "id" : "https://d-nb.info/gnd/7749153-1"
+      },
+      "id" : "https://d-nb.info/gnd/4262309-1",
+      "gndIdentifier" : "4262309-1",
+      "altLabel" : [ "Insektenforscher" ]
+    } ]
+  }, {
+    "type" : [ "ComplexSubject" ],
+    "label" : "Wissenschaftler",
+    "componentList" : [ {
+      "type" : [ "SubjectHeading" ],
+      "label" : "Wissenschaftler",
+      "source" : {
+        "label" : "Gemeinsame Normdatei (GND)",
+        "id" : "https://d-nb.info/gnd/7749153-1"
+      },
+      "id" : "https://d-nb.info/gnd/4066567-7",
+      "gndIdentifier" : "4066567-7"
+    } ]
+  } ],
+  "spatial" : [ {
+    "id" : "https://nwbib.de/spatial#Q2942",
+    "label" : "Solingen",
+    "type" : [ "Concept" ],
+    "source" : {
+      "id" : "https://nwbib.de/spatial",
+      "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
+    },
+    "focus" : {
+      "id" : "http://www.wikidata.org/entity/Q2942",
+      "label" : "Solingen",
+      "type" : [ "http://www.wikidata.org/entity/Q85635630", "http://www.wikidata.org/entity/Q1549591", "http://www.wikidata.org/entity/Q106517174", "http://www.wikidata.org/entity/Q707813", "http://www.wikidata.org/entity/Q42744322" ],
+      "geo" : {
+        "lat" : "51.166666666",
+        "lon" : "7.083333333"
+      }
+    }
+  } ],
+  "subjectslabels" : [ "Meigen, Johann Wilhelm", "Solingen", "Wissenschaftler", "Entomologe" ],
+  "hasItem" : [ {
+    "id" : "http://lobid.org/items/990110486750206441:DE-Sol1:ZDB50011048675-000010#!",
+    "callNumber" : "MA 3264/[1]",
+    "inventoryNumber" : "1993.373",
+    "inCollection" : [ {
+      "id" : "http://lobid.org/organisations/DE-Sol1#!",
+      "label" : "Stadtarchiv Solingen, Bibliothek"
+    } ],
+    "heldBy" : {
+      "isil" : "DE-Sol1",
+      "id" : "http://lobid.org/organisations/DE-Sol1#!",
+      "label" : "DE-Sol1#!"
+    },
+    "label" : "Stadtarchiv Solingen, Bibliothek",
```

Comment:
I think, we should use another label here. AFAICS, we are currently using in lobid-resources "lobid bestandsressource" in the string at this position. We could use the same here of – if we want to mark the difference of the source data (Aleph/Strapi instead of Alma) also in the `label`– something "DE-Sol1-Bestandsressource" or "Exemplar von Stadtarchiv Solingen, Bibliothek"  o.ä.

Reply:
Or was this label intended to be in `hasItem.heldBy.label` where we now have "DE-Sol1#!`? This would make sense..

Reply:
I fixed this by fixing the fix: https://github.com/hbz/lobid-extra-holdings/commit/7de5d028f176c3cab8811acb5183652b41b0aea7 and uploading the newly generated upload file: https://github.com/hbz/lobid-extra-holdings/commit/579842349ac69d53feb7b197da2607b13c7c735b